src/PROPOSAL/detail/PROPOSAL/scattering/stochastic_deflection/bremsstrahlung/BremsGinneken.cxx

```diff
@@ -15,11 +15,10 @@ double stochastic_deflection::BremsGinneken::f_nu_g(double nu, double n, double
 }
 
 double stochastic_deflection::BremsGinneken::get_nu_g(double E, double n, double k_4) const {
-    auto f = [this, &n, &k_4](double nu) {
+    auto f = [&, n, k_4](double nu) {
```

Comment:
Since we only want to capture n and k_4 here, can't we also just write
```
auto f = [=](double nu) {
```
or what is the reason that it is written this way?

Reply:
I have just tested it and it should work. Is there a difference between 
`auto f = [=](double nu) {`
and
`auto f = [&](double nu) {` ? 
Both seem to work.

Reply:
If you use `&`, all variables are captured by reference.
If you use `=`, all variables are captured by value.
See [here](https://docs.microsoft.com/en-us/cpp/cpp/lambda-expressions-in-cpp?view=msvc-170#capture-clause) for example.

Reply:
I was just a bit confused about the syntax here, but it shouldn't make a difference in this code since all variabled are onl read inside the lambda expression 😄